docs/dev-guides/howto-release-helm-chart.md

```diff
@@ -8,6 +8,7 @@
   * [Publish the Helm Chart](#publish-the-helm-chart)
     * [Create a Helm Chart Package](#create-a-helm-chart-package)
     * [Publish to `https://azureiiot.blob.core.windows.net/helm` Repository](#publish-to-httpsazureiiotblobcorewindowsnethelm-repository)
```

Comment:
is this still valid?

Reply:
Yes, we still have the repo in Storage Account under same URL.